codenav/src/codenavigation.h

```diff
@@ -42,7 +42,6 @@
 
 /* Last */
 #include "plugindata.h"		/* this defines the plugin API */
-#include "geanyfunctions.h"	/* this wraps geany_functions function pointers */
```

Comment:
Should change the code to use `g_strcmp0` or `strcmp` instead of using the plugin API as a redundant utility library :)